installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -44,6 +44,28 @@ vagrant_dev_box_add() {
     fi
 }
 
+vagrant_dev_first_up() {
+    declare os="$1"
```

Comment:
I don't think quotes are necessary

Reply:
Same question as https://github.com/radiasoft/download/pull/444#discussion_r1157801266

Reply:
I would say one thing: we need better tests to give confidence that things are working correctly. In Bash, it's pretty easy to mock things, e.g. add a command in the path that gets used instead of `vagrant`, for example.

In an ideal world, we'd program everything in a type safe language, and it would be as easy to develop as it is with bash, but that world doesn't exist. The nature of shells is that they are untyped. 

That's why I think testing is the only way to convince ourselves that the program is running correctly.